openg2p_disbursement/wizard/batch_from_selected_beneficiary.py

```diff
@@ -29,6 +29,14 @@ class RegisterWizard(models.TransientModel):
         required=True,
     )
     use_active_domain = fields.Boolean('Use active domain')
+    selected_record_count = fields.Integer('Selected Record Count', compute="_compute_selected_record")
+    all_record_count = fields.Integer('All Record Count', compute="_compute_selected_record")
+
+
+    @api.depends('use_active_domain')
+    def _compute_selected_record(self):
+        self.all_record_count = self.env['openg2p.beneficiary'].search_count([])
```

Comment:
@atularvind use should be passing active_domain like I do in the code to filter beneficiaries to act on